fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -905,6 +908,10 @@ private boolean tryProcessProfileAsync(ExecPlan plan, int retryIndex) {
             }
             QeProcessorImpl.INSTANCE.unMonitorQuery(executionId);
             QeProcessorImpl.INSTANCE.unregisterQuery(executionId);
+            if (Config.enable_collect_query_detail_info && Config.enable_profile_log) {
+                String jsonString = GSON.toJson(queryDetail);
+                PROFILE_LOG.info(jsonString);
```

Comment:
How to read the profile written to the disk? Can we get these profiles from current get profile interface?

Reply:
this feature is asked by sass team. they want to get profile file from disk and resovle it by theyselves so it won't lost. Any other solution is not as stable as they want